css/styles.css

```diff
@@ -299,20 +293,19 @@ a {
 }
 
 a.studio-image {
-    background: url(../images/two_rockets.png); /* Путь к файлу с исходным  рисунком */
-    display: block; /* Ссылка как блочный элемент */
-    width: 35px; /* Ширина рисунка в пикселах */
-    height: 58px; /* Высота рисунка */
+    background: url(../images/twoRockets.png); 
+    display: block; 
+    width: 35px; 
+    height: 58px; 
 }
 
 a.studio-image:hover {
-    background-position: 0 -61px; /* Смещение фона */
+    background-position: 0 -61px; 
 }
 /* -------------studio end--------------- */
```

Comment:
You already have comment ` /* --------development------------ */` which means that `studio` is end